### PR TITLE
[Merged by Bors] - chore(NumberTheory/Numberfield/*): rename ringEquiv_complex_of_isComplex -> ringEquivComplexOfIsComplex

### PR DESCRIPTION
**Pull request by kbuzzard**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

Suggested in a [reviewer comment](https://github.com/leanprover-community/mathlib4/pull/19646/files#r1864687306) in #19646 

This PR makes the following renames (and deprecates the old defs):

`def extensionEmbedding_of_isReal` -> `def extensionEmbeddingOfIsReal`
`def ringEquiv_complex_of_isComplex` -> `def ringEquivComplexOfIsComplex`
`def isometryEquiv_complex_of_isComplex` -> `def isometryEquivComplexOfIsComplex`
`def ringEquiv_real_of_isReal` -> def `ringEquivRealOfIsReal`
`def isometryEquiv_real_of_isReal` -> def `isometryEquivRealOfIsReal`

### CODE REVIEW

File: Mathlib/NumberTheory/NumberField/Completion.lean
Comment (Ruben-VandeVelde) [inline Mathlib/NumberTheory/NumberField/Completion.lean:145]: Same here, btw

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge 

Thanks!

**Comment by riccardobrasca** [comment]
I think you can follow Ruben's comment and change `isometryEquiv_complex_of_isComplex` in this PR. Thanks!

bors d+

**Comment by urkud** [comment]
BTW, there are more `*_*` definitions in this file.

File: Mathlib/NumberTheory/NumberField/Completion.lean
Comment (kbuzzard) [reply Mathlib/NumberTheory/NumberField/Completion.lean]: Fixed. Thanks!

**Comment by kbuzzard** [comment]
Yes Yury's right, there are three more!

Summary of this PR currently: 

`def extensionEmbedding_of_isReal` -> `def extensionEmbeddingOfIsReal`
`def ringEquiv_complex_of_isComplex` -> `def ringEquivComplexOfIsComplex`
`def isometryEquiv_complex_of_isComplex` -> def isometryEquivComplexOfIsComplex`
`def ringEquiv_real_of_isReal` -> `def ringEquivRealOfIsReal`
`def isometryEquiv_real_of_isReal` -> `def isometryEquivRealOfIsReal`

Can someone check to see if I've missed any more before I merge?

File: Mathlib/NumberTheory/NumberField/Completion.lean
Comment (kbuzzard) [inline Mathlib/NumberTheory/NumberField/Completion.lean:174]:
```suggestion
/-- If `v` is a real infinite place, then the embedding `v.Completion →+* ℝ` is surjective. -/
```

File: Mathlib/NumberTheory/NumberField/Completion.lean
Comment (kbuzzard) [inline Mathlib/NumberTheory/NumberField/Completion.lean:193]:
```suggestion
/-- If the infinite place `v` is real, then `v.Completion` is isometric to `ℝ`. -/
```

**Comment by kbuzzard** [comment]
I think I'm just being over-cautious. I checked the file again after sleeping on it, and I'm pretty sure that it's now OK.

bors r+

**Pull request closed without merging**
